eogrow/utils/testing.py

```diff
@@ -305,8 +306,10 @@ def run_config(
     *,
     output_folder_key: Optional[str] = None,
     reset_output_folder: bool = True,
```

Comment:
I think it would make more sense if this `reset_output_folder` would run only in the beginning, not between each config. If a test config chain is run, you're usually not interested in deleting intermediate steps. And in case you are, then you could just split the test in separate runs to achieve the same, which makes more sense anyway.

Reply:
i actually think only the last one should be reset, since it might mess with the content tester

Reply:
Agreed to leave as is.